database/migrations/2021_12_09_171903_create_tours_table.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateToursTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('tours', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('guest_house_id')
+            ->constrained()
+            ->cascadeOnUpdate()
+            ->restrictOnDelete();
+            $table->foreignId('location_id')
+            ->constrained()
+            ->cascadeOnUpdate()
+            ->restrictOnDelete();
+            $table->foreignId('sport_activity_id')
```

Comment:
maybe we should accept it as nullable?

Reply:
If I understand correctly, this solves the problem of the obligation to attend a sports activity on the tour, added nullable